src/main/java/org/kohsuke/github/GHPersonSet.java

```diff
@@ -10,6 +10,8 @@
  * @author Kohsuke Kawaguchi
  */
 public class GHPersonSet<T extends GHPerson> extends HashSet<T> {
+    private static final long serialVersionUID = 1L;
+ 
```

Comment:
Next time when adding `serialVersionUID`s, I recommend running `serialver` and take the version UID that would have been generated anyway.